src/sites/link/linkdrop.net.js

```diff
@@ -268,6 +264,13 @@
       o.observe(b, {
         attributes: true,
       });
```

Comment:
We can remove `MutationObserver` because now we are going to use a timer to check the button.
`MutationObserver` might make it faster but I don't think it makes that much differences.
So just remove it to keep the simplicity.